### PR TITLE
Update release notes for 0.123

### PR DESCRIPTION
**Pull request by haozhun**

### CODE REVIEW

**Comment by dain** [comment]
Add a warning that `DELETE` currently only works for tables where all partition keys are `VARCHAR`

File: presto-docs/src/main/sphinx/release/release-0.123.rst [hunk 1]
```diff
@@ -16,6 +16,7 @@ General Changes
 * Fix issue with :func:`histogram` that can cause failures or incorrect results
   when there are more than ten buckets.
 * Optimize execution of cross join.
+* Run Presto server as presto user in RPM init.d scripts.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.123.rst]:
```
* Run Presto server as ``presto`` user in RPM init scripts.

File: presto-docs/src/main/sphinx/release/release-0.123.rst [hunk 2]
```diff
@@ -46,3 +47,7 @@ entire partitions.  For example, to delete from the above table, execute the fol
 
     DELETE FROM orders
     WHERE order_date = '2015-10-15' AND order_region = 'APAC'
+
+.. note::
+    * Only partition keys of VARCHAR type is supported temporarily
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.123.rst]: Add newline before list

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.123.rst]: List items should be full sentences, so end in period

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.123.rst]: "Currently, only partition keys of type VARCHAR are supported."

File: presto-docs/src/main/sphinx/release/release-0.123.rst [hunk 3]
```diff
@@ -46,3 +47,7 @@ entire partitions.  For example, to delete from the above table, execute the fol
 
     DELETE FROM orders
     WHERE order_date = '2015-10-15' AND order_region = 'APAC'
+
+.. note::
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.123.rst]:
```
.. note::

    Currently, Hive deletion is only supported for partitioned tables.
    Additionally, partition keys must be of type VARCHAR.

**Pull request merged**